tool/go.mod

```diff
@@ -1,6 +1,6 @@
-module github.com/launchdarkly/sdk-meta
+module github.com/launchdarkly/sdk-meta/tool
 
-go 1.22.0
```

Comment:
Gonfalon is on a lower Go version, and having this be 1.22 would require them to update.. a pain.